source/TLS.rst

```diff
@@ -0,0 +1,181 @@
+.. _tls:
+
+Transport Layer Security (TLS)
+******************************
+
+The Percona Distribution for PostgreSQL Operator uses Transport Layer Security
+(TLS) cryptographic protocol for the following types of communication:
+
+* Internal - communication between PostgreSQL instances in the cluster
+* External - communication between the client application and the cluster
+
+The internal certificate is also used as an authorization method.
```

Comment:
mm, not sure. Authorization method for what?

Reply:
fixed